.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - run: echo > passfile
       - run: ./parity --chain 2>&1 &
```

Comment:
@hswick  you can start parity with `/parity --jsonrpc-interface all --no-warp`. It starts ipc and http servers
`